packages/api/src/client/ApiClient.ts

```diff
@@ -31,10 +31,15 @@ import {
   ChangeUserPassword,
   ChangeUserDisplayName,
   SystemModels,
+  ExternalClientPayload,
+  RetrieveExternalLoginPayload,
+  ExternalLoginPayload,
 } from '../interfaces';
 import { ChangeUserAvatar } from '../interfaces/change-user-avatar.interface';
 import { BulkDeleteNavigationEntriesData } from '../interfaces/navigation-entry.interface';
 
+let fetchRetries = 0;
```

Comment:
@dcoscolla-intermedia Shouldn't this be an optional parameter of the securedFetch method with a fallback value?